src/pages/gateway/migration.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Migration
+description: This page describes breaking changes and how to migrate from one version of API Mesh for Adobe Developer App Builder to another.
+---
+
+# Migration
+
+This page lists breaking changes between versions of API Mesh for Adobe Developer App Builder. For a list of new features and resolved issues, refer to the [release notes](release-notes.md).
+
+To update to the latest version, run the following command:
+
+```bash
+npm install -g @adobe/aio-cli-plugin-api-mesh
+```
+
+See our [NPM page](https://www.npmjs.com/package/@adobe/aio-cli-plugin-api-mesh) or [Getting Started](getting-started.md#configure-your-environment) for more information.
+
+## Migrating from Beta to GA
+
+If you participated in the beta for API Mesh, you may need to account for the following breaking changes when updating to the GA release.
```

Comment:
if you have a mesh in beta version (or participated) in beta program 
i am not sure if we want to disclose the beta program in the docs...

Reply:
I think All meshes have to be recreated - whether their config are be compatible with GAA or not
If that's the case - that's an action item that we have to state